src/common/mainHelpers.mo

```diff
@@ -0,0 +1,182 @@
+import Types "../modclub/types";
```

Comment:
just curiours: Is moving these code from main.mo necessary for your PR, or are you just performing code refactoring within the same PR?